### PR TITLE
build viewer

### PR DESCRIPTION
**Pull request by costin555**
### PR Checklist

- [ ] Brief description of changes
- [ ] Links to any relevant issues
- [ ] Required status checks are passing
- [ ] User cases if changes impact the user's experience
- [ ] `@mention` a maintainer to request a review

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

File: platform/viewer/src/index.js [hunk 1]
```diff
@@ -25,18 +25,26 @@ import OHIFMeasurementTrackingExtension from '@ohif/extension-measurement-tracki
...
+//   ],
+// };
 
 /** Create App */
-const app = React.createElement(App, appProps, null);
+// const app = React.createElement(App, appProps, null);
 
 /** Render */
-ReactDOM.render(app, document.getElementById('root'));
+// ReactDOM.render(app, document.getElementById('root'));
```
Comment (shivasuri) [inline platform/viewer/src/index.js:42]: Can we delete instead of comment out?

Comment (costin555) [reply platform/viewer/src/index.js:42]: Yes, we can delete them

Comment (costin555) [reply platform/viewer/src/index.js:42]: But I think we should leave it to provide faster dev speed.

Comment (danihodovic) [reply platform/viewer/src/index.js:42]: Why do the commented lines have faster dev speed? Are they used as some sort of reference or documentation?

**Pull request closed without merging**